src/components/ui/TaskFooter.jsx

```diff
@@ -28,5 +38,6 @@ export default function TaskFooter({ isCompleted = false }) {
 }
 
 TaskFooter.propTypes = {
-  isCompleted: propTypes.bool
+  isCompleted: propTypes.bool,
+  taskId: propTypes.number
```

Comment:
If you are not using defaultProps then must use isRequired else there is an opportunity to occur unhandled bug.
```suggestion
  isCompleted: propTypes.bool.isRequired,
  taskId: propTypes.number.isRequired
```

Reply:
Hi @iud-wtag bhai, `isCompleted ` is set to `false` by default in the functional component argument. And I have made `taskId` required for this component. thanks